### ISSUE TITLE
Rule proposal: `prefer-blob-reading-methods`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1269]
[`FileReader`](https://developer.mozilla.org/en-US/docs/Web/API/FileReader) is kind of old, and it doesn't use promise. A better way to read file should be ["blob reading methods"](https://github.com/w3c/FileAPI/pull/117)

## Fail

```js
const arrayBuffer = await new Promise((resolve, reject) => {
	const fileReader = new FileRead();
	fileReader.onload = () => resolve(fileReader.result)
	fileReader.onerror = () => reject(fileReader.error)
	fileReader.readAsArrayBuffer(blob);
});
```

```js
const arrayBuffer = await new Response(blob).arrayBuffer();
```

## Pass

```js
const arrayBuffer = await blob.arrayBuffer();
```

**Comment by sindresorhus** [comment]
This is now accepted.

**Comment by jimmywarting** [comment]
most often when you want to display a simple preview from a file input then it's faster to just display a blob:url from using `URL.createObjectURL` (as dataURL takes time to encode/decode and uses more memory)

but you always need to be carful of when you are not revoking object urls
